### PR TITLE
Nodetask

### PR DESCRIPTION
**Pull request by horaanchal**
Perform CRUD on previous dashboard

### CODE REVIEW

File: Anchal_Hora/nodejs/Login.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+function func(){
```
Comment (vishivish18) [inline Anchal_Hora/nodejs/Login.js]: `func` is not a proper name for function

File: Anchal_Hora/nodejs/Login.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    window.location.href = "./admin/index.html";
+    }
+document.getElementById("button").addEventListener("click",Gettoken)
+function Gettoken(){
```
Comment (vishivish18) [inline Anchal_Hora/nodejs/Login.js]: use consistent casing of variables and function names, use all lower case

File: Anchal_Hora/nodejs/Login.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    var Sno=parseInt(document.getElementById("inputid").value);
+    var Username=document.getElementById("inputName").value;
+    var Email=document.getElementById("inputEmail").value;
+    var Password=document.getElementById("inputPassword").value;
+    
+    
+    var get = {
```
Comment (vishivish18) [inline Anchal_Hora/nodejs/Login.js]: `get` is not the right name for the request object

File: Anchal_Hora/nodejs/UI/create.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (vishivish18) [inline Anchal_Hora/nodejs/UI/create.html:2]: why are you not using Bootstrap ?

File: Anchal_Hora/nodejs/UI/js/delete.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+document.getElementById("buttondelete").addEventListener("click",admindeletedata)
+function admindeletedata(){
```
Comment (vishivish18) [inline Anchal_Hora/nodejs/UI/js/delete.js]: when you have multiple words use camelCase

**Comment by vishivish18** [comment]
@horaanchal you need work on your code, you need to take care of your naming convention, for files, functions and variables. Please go through your whole code again and then raise a PR'
Let me know if you have any doubts

**Comment by vishivish18** [comment]
@horaanchal Also why your Node PR contains code for C#?

**Comment by horaanchal** [comment]
> @horaanchal Also why your Node PR contains code for C#?

That I di

> @horaanchal you need work on your code, you need to take care of your naming convention, for files, functions and variables. Please go through your whole code again and then raise a PR'
> Let me know if you have any doubts

Sure I will revise the whole code once and let you know.

**Comment by horaanchal** [comment]
> @horaanchal Also why your Node PR contains code for C#?

That I did unintentionally, I will fix it.

File: Anchal_Hora/nodejs/Login.js (hunk 1 above)
Comment (horaanchal) [reply Anchal_Hora/nodejs/Login.js]: function name is now login

File: Anchal_Hora/nodejs/UI/js/delete.js (hunk 5 above)
Comment (horaanchal) [reply Anchal_Hora/nodejs/UI/js/delete.js]: Used camelCase Convention

File: Anchal_Hora/nodejs/Login.js (hunk 2 above)
Comment (horaanchal) [reply Anchal_Hora/nodejs/Login.js]: > @horaanchal you need work on your code, you need to take care of your naming convention, for files, functions and variables. Please go through your whole code again and then raise a PR'
> Let me know if you have any doubts

Sure I will go through my code once again and let you know.

Comment (horaanchal) [reply Anchal_Hora/nodejs/Login.js]: > use consistent casing of variables and function names, use all lower case

Proper naming convention followed.

File: Anchal_Hora/nodejs/Login.js (hunk 3 above)
Comment (horaanchal) [reply Anchal_Hora/nodejs/Login.js]: Changed the name into json

File: Anchal_Hora/nodejs/UI/create.html (hunk 4 above)
Comment (horaanchal) [reply Anchal_Hora/nodejs/UI/create.html:2]: Using bootstrap in the main html file that is index.html

**Pull request closed without merging**